<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrushPenSelection</class>
 <widget class="QDialog" name="BrushPenSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>493</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="1" column="0" rowspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Brush settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Brush alpha</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QComboBox" name="brushStyle">
        <item>
         <property name="text">
          <string>No Brush</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Solid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dense 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dense 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dense 3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dense 4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dense 5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dense 6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dense 7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Horizontal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vertical</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cross pattern</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Backwards diagonal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Forwards diagonal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Diagonal cross</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="brushColor">
        <property name="text">
         <string>Choose brush color</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="alphaSpin">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QSlider" name="alphaSlider">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="pathItemHasBrush">
        <property name="text">
         <string>Path item has brush</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Pen settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QSlider" name="widthSlider">
        <property name="maximum">
         <number>2500</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="penColor">
        <property name="text">
         <string>Choose pen color</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="cosmetic">
        <property name="text">
         <string>Cosmetic</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Pen alpha</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="widthSpinner"/>
      </item>
      <item row="6" column="0">
       <widget class="QSlider" name="penAlphaSlider">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="penAlphaSpin">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Arrow settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Arrow width and height</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QDoubleSpinBox" name="arroww">
        <property name="prefix">
         <string notr="true">w: </string>
        </property>
        <property name="suffix">
         <string notr="true">px</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="arrowh">
        <property name="prefix">
         <string notr="true">h: </string>
        </property>
        <property name="suffix">
         <string notr="true">px</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Blur settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0" colspan="2">
       <widget class="QSlider" name="radSlider">
        <property name="maximum">
         <number>3000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QCheckBox" name="performance">
        <property name="text">
         <string>Performance Hint</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QCheckBox" name="animated">
        <property name="text">
         <string>Animated Hint</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="quality">
        <property name="text">
         <string>Quality Hint</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="whatsThis">
         <string notr="true">http://doc.qt.io/qt-5/qgraphicsblureffect.html#BlurHint-enum</string>
        </property>
        <property name="text">
         <string>&lt;a href=&quot;http://doc.qt.io/qt-5/qgraphicsblureffect.html#BlurHint-enum&quot;&gt;Blur Hints</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextBrowserInteraction</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Blur Radius</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="radSpinner">
        <property name="suffix">
         <string notr="true">px</string>
        </property>
        <property name="maximum">
         <double>30.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>alphaSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>alphaSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alphaSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>alphaSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>penAlphaSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>penAlphaSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>penAlphaSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>penAlphaSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
